src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -69,6 +80,9 @@ public IActionResult AddMultipleAnswerQuestion([FromBody] MultipleQuestionAddMod
         [HttpPut]
         public IActionResult EditMultipleAnswerQuestion([FromBody] MultipleQuestionEditModel multipleAnswersQuestionDTO)
         {
+            IEnumerable<string> answers = multipleAnswersQuestionDTO.QuestionAnswers.Select(a => a.AnswersText);
+            _logger.LogInformation($"QuestionText: {multipleAnswersQuestionDTO.QuestionText}|Answers: {String.Join(", ", answers)}");
```

Comment:
The line 84 is duplicated 69's. The candidate to be extracted as a separate method.